### ISSUE TITLE
nuxt3 not working on windows

### ISSUE THREAD

**Issue by userquin** [nuxt/framework#1793]
### Environment

```shell
npx nuxi info
Nuxt CLI v3.0.0                                                                                                                                                                                          09:10:54
RootDir: F:\work\projects\quini\GitHub\nuxt-framework                                                                                                                                                    09:10:55
Nuxt project info:                                                                                                                                                                                       09:10:55

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.12.0`
- Nuxt Version:     `3.0.0`
- Package Manager:  `yarn@3.1.0`
- Bundler:          `Vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------
```

### Describe the bug

The main problem are absolute paths on `vite` plugin.

~~Cloning this repo on windows and just running `yarn build && yarn play` result on error.~~

### Reproduction

Testing `nuxt3` on `vitesse nuxt3` on windows fails: https://github.com/antfu/vitesse-nuxt3/issues/4

Clone `vitesse nuxt3` repo on a windows machine, running `pnpm run dev` and opening app on browser will fail.

~~Clone this repo in a windows machine: `yarn && yarn dev`, open browser and you get the error on logs.~~

### Additional context

### Logs

```shell
yarn play
Nuxt CLI v3.0.0                                                                                                                                                                                          09:07:37
                                                                                                                                                                                                         09:07:37
  > Local:    http://localhost:3000/
  > Network:  http://172.26.0.1:3000/
  > Network:  http://192.168.1.37:3000/

i Vite warmed up in 221ms                                                                                                                                                                                09:07:38
√ Vite server built in 441ms                                                                                                                                                                             09:07:39
√ Nitro built in 154 ms                                                                                                                                                                                  09:07:39
Cannot read properties of undefined (reading 'map')
  at Module.normalizePlugins (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:177:28)
  at $id_63620399 (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:28:39)
  at async __instantiateModule__ (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:815:3)
Cannot read properties of undefined (reading 'map')
  at Module.normalizePlugins (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:177:28)
  at $id_63620399 (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:28:39)
  at async __instantiateModule__ (file:///F:/work/projects/quini/GitHub/nuxt-framework/playground/.nuxt/dist/server/server.mjs:815:3)
```

**Comment by userquin** [comment]
Seems I get it working on `playground`, but on `vitesse nuxt3` repo there is a problem with the virtual module (vite) and absolute paths.

**Comment by userquin** [comment]
I get it working adding the `file:///` protocol on `load`, see https://github.com/antfu/vitesse-nuxt3/issues/4#issuecomment-963936017

**Comment by userquin** [comment]
fixed on #1779
